config/04-kubernetes/42-hub-preview-redirect-ingress.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: ui
+  namespace: tekton-hub-preview
+  annotations:
+    acme.cert-manager.io/http01-edit-in-place: 'true'
+    cert-manager.io/cluster-issuer: letsencrypt-prod
+    nginx.ingress.kubernetes.io/permanent-redirect: https://hub.tekton.dev
+    nginx.ingress.kubernetes.io/force-ssl-redirect: 'true'
+spec:
+  tls:
+    - secretName: ui-hub-preview-tekton-dev-tls
+      hosts: [hub-preview.tekton.dev]
+  rules:
+    - http:
```

Comment:
I usually include `host: hub-preview.tekton.dev` as key under `rules` too, but I'm not 100% sure if it's required:

```
spec:
  rules:
  - host: hub-preview.tekton.dev
     http: (...)
```

Reply:
Sorry for late reply 
Is it fine if we apply the ingress on dogfooding first and see and if that works fine then merge the pr ??